pranav/cart/main.rb

```diff
@@ -0,0 +1,192 @@
+class Cart
+    def initialize
+      @menu = { 1 => { name: 'aa', price: 10, quantity: 10 }, 2 => { name: 'bb', price: 30, quantity: 10 }, 3 => { name: 'cc', price: 40, quantity: 7 }, 4 => { name: 'dd', price: 70, quantity: 10 }, 5 => { name: 'ee', price: 90, quantity: 10 } }
```

Comment:
Rather than giving index as keys wouldnt it be more convinient to give item name as a key

Reply:
Right and ID is mostly the unique attribute which usually is name and not index